sharedData/parameterBuckets.json

```diff
@@ -0,0 +1,121 @@
+﻿{"intrinsicParameters" : [
+  "a_1",
+  "a_2",
+  "chi_1_in_plane",
+  "chi_2_in_plane",
+  "chi_eff",
+  "chi_p",
+  "chirp_mass",
+  "chirp_mass_source",
+  "cos_tilt_1",
+  "cos_tilt_2",
+  "mass_1",
+  "mass_1_source",
+  "mass_2",
+  "mass_2_source",
+  "mass_ratio",
+  "phase",
+  "phi_1",
+  "phi_12",
+  "phi_2",
+  "phi_jl",
+  "psi",
+  "spin_1x",
+  "spin_1y",
+  "spin_1z",
+  "spin_2x",
+  "spin_2y",
+  "spin_2z",
+  "symmetric_mass_ratio",
+  "theta_jn",
+  "tilt_1",
+  "tilt_2",
+  "total_mass",
+  "total_mass_source"
+],
+
+"extrinsicParameters" : [
+  "H1_time",
+  "azimuth",
+  "comoving_distance",
+  "dec",
+  "geocent_time",
+  "iota",
+  "luminosity_distance",
+  "ra",
+  "redshift",
+  "zenith"
+],
+
+"otherParameters" : [
```

Comment:
The intention was to provide an example of what bilby outputs would be categorized as other. I actually removed it myself at one point but I think Venura added it back.

Reply:
It's ok for it to be there as an example, but then you need to say in the documentation that it's just an example and not exhaustive

Reply:
Added comment on frontend to clarify

Reply:
```suggestion
"otherParametersSample" : [
```
Might be a good idea to rename `otherParameters` here to something like `otherParametersSample`, just so that it's clear to anyone looking that this isn't expected to be the exact list

Reply:
Updated this, is chnaged across rabit-frontend aswell